samples/echo/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        applicationId 'com.google.sample.oboe.echo'
+        minSdkVersion 24
+        targetSdkVersion 27
+        versionCode 1
+        versionName '1.0'
+        ndk {
```

Comment:
x86 build variants are useful when testing on emulator

Reply:
got that part, uploaded with the change